tests/Integration/Infrastructure/Controller/LogoutControllerTest.php

```diff
@@ -21,7 +21,7 @@ public function testLogout(): void
         $crawler = $client->followRedirect();
         $this->assertRouteSame('app_landing');
 
-        $enterLink = $crawler->selectLink("Participer à l'expérimentation");
+        $enterLink = $crawler->selectLink(value: "Participer à l'expérimentation");
```

Comment:
Toi aussi tu as ces hints qui apparaissent avec la dernière version de PHP for VSCode ?

Appliquer ces hints ça génère du bruit dans les PR je trouve... par exemple ici comme il y a un seul argument c'est pas très utile d'utiliser un keyword argument. Perso je les ai désactivés... Le setting c'est `php.inlayHints.parameters.enabled`

Reply:
Oui ! C'est très relou ! Merci pour le tips :+1: